sc/shared/src/test/scala/sigmastate/ScriptVersionSwitchSpecification.scala

```diff
@@ -266,44 +266,45 @@ class ScriptVersionSwitchSpecification extends SigmaDslTesting {
 
   /** Rule#| BlockVer | Block Type| Script Version | Release | Validation Action
     * -----|----------|-----------|----------------|---------|--------
-    * 19   | 4        | candidate | Script v3      | v5.0    | skip-accept (rely on majority)
-    * 20   | 4        | mined     | Script v3      | v5.0    | skip-accept (rely on majority)
+    * 19   | 4        | candidate | Script v3      | v6.0    | R6.0-JIT-verify
+    * 20   | 4        | mined     | Script v3      | v6.0    | R6.0-JIT-verify
     */
   property("Rules 19,20 | Block v4 | candidate or mined block | Script v3") {
     forEachActivatedScriptVersion(activatedVers = Array[Byte](3)) // version for Block v4
     {
-      forEachErgoTreeVersion(ergoTreeVers = Array[Byte](3, 4)) { // scripts >= v3
+      forEachErgoTreeVersion(ergoTreeVers = Array[Byte](3)) { // scripts >= v3
```

Comment:
same is here, versions 3,4 in v5.x means versions higher that MaxSupportedVersion, so should be 4,5